packages/bootstrap/extra/dcos_internal_utils/cli.py

```diff
@@ -62,6 +66,27 @@ def noop(b, opts):
 }
 
 
+def _apply_sysctl(setting, service):
+    try:
+        check_call(["sysctl", "-q", "-w", setting])
+    except CalledProcessError:
```

Comment:
Actually, we do not know the exact versions / OS offhand. There are versions of the kernel for which some settings are applicable and for some it is not. Either we will have to define the known list of kernels versions that DCOS will support and then apply against it, or we probe and apply. Even probing and applying is similar to applying and not failing as it is done here.

@spahl can chime in here with what might we want to do.